litmus-portal/backend/graphql-server/pkg/database/mongodb/mongodb.go

```diff
@@ -13,95 +13,82 @@ import (
 )
 
 var clusterCollection *mongo.Collection
-var backgroundContext context.Context
+var backgroundContext = context.Background()
 
 //DBInit initializes database connection
-func DBInit() {
-	dbServer := os.Getenv("DB_SERVER")
+func DBInit() error {
+	dbServer := os.Getenv("MONGODB_SERVER")
 	dbName := os.Getenv("DB_NAME")
 	collectionName := os.Getenv("COLLECTION_NAME")
-	clientOptions := options.Client().ApplyURI("mongodb://" + dbServer)
-	client, err := mongo.Connect(context.TODO(), clientOptions)
 
+	clientOptions := options.Client().ApplyURI("mongodb://" + dbServer)
+	client, err := mongo.Connect(backgroundContext, clientOptions)
 	if err != nil {
-		log.Fatal(err)
```

Comment:
Need to print the error in some specific area. Otherwise, return the error

Reply:
This is a fatal error, so we need to panic in this case, we can do this right here or handle it in the main func.

Reply:
https://github.com/litmuschaos/litmus/pull/1815#discussion_r465891708